sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MyApplication.java

```diff
@@ -16,13 +16,19 @@ public void onCreate() {
     // Example how to initialize the SDK manually which allows access to SentryOptions callbacks.
     // Make sure you disable the auto init via manifest meta-data: io.sentry.auto-init=false
     // SentryAndroid.init(
-    // this,
-    // options -> {
-    //   options.setBeforeSend(event -> {
-    //     event.setTag("sample-key", "before-send");
-    //   });
-    //   options.setAnrTimeoutIntervalMillis(2000);
-    // });
+    //    this,
+    //    options -> {
+    //      options.setBeforeSend(
+    //          (event, hint) -> {
+    //            event.setTag("sample-key", "before-send");
+    //            return event;
+    //          });
+    //      options.setAnrTimeoutIntervalMillis(2000);
+    //
+    // options.setDsn("https://d3964e5d568641c484c536600fb433a1@o552180.ingest.sentry.io/5677396");
```

Comment:
even if initing manually, it still reads the values from manifest, so if you have your `dsn` set in there, you don't actually need it here, it does not hurt being here either

Reply:
gotcha, thanks! I think I'll remove it to make things simpler.

Reply:
Removed in 8090b9e7297431492d416dc72944967cb98a7c58